.env.example

```diff
@@ -1,7 +1,14 @@
 # The following values are only necessary for deployment.
 # Contact the project admin to get them if you want to deploy some changes to
 # the production server
+
+# for deploying:
 PRODUCTION_SERVER=xxx.xxx.xxx.xxx
 DEPLOY_USER=some_user
 APP_PATH=/path/to/app
 BRANCH=vcs_branch
+
+# used by the system:
```

Comment:
sure.  Can you just prompt me with a sentence of phrase to use?  I'm kind of steeped in this now and so can't quite come up with something that seems helpful.  Maybe we should just take from what we say in the wiki? https://github.com/AgileVentures/shf-project/wiki/Production-server#environment-variables